### PR TITLE
Fix stub for gatk4/reblockgvcf and gatk4/genomicsdbimport

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
LGTM

File: modules/nf-core/gatk4/genomicsdbimport/main.nf [hunk 1]
```diff
@@ -70,6 +70,7 @@ process GATK4_GENOMICSDBIMPORT {
 
     genomicsdb_command = "--genomicsdb-workspace-path ${prefix}"
     interval_command = interval_file ? "--intervals ${interval_file}" : "--intervals ${interval_value}"
+    updated_db = []
```
Comment (muffato) [inline modules/nf-core/gatk4/genomicsdbimport/main.nf:73]: Shouldn't this line be copied to the `script:` section too ? It feels like `updated_db` should also have a default value there

Comment (nvnieuwk) [reply modules/nf-core/gatk4/genomicsdbimport/main.nf]: Done!

**Comment by muffato** [comment]
@nvnieuwk : the gatk4/genomicsdbimport tests are failing

**Comment by nvnieuwk** [comment]
> @nvnieuwk : the gatk4/genomicsdbimport tests are failing


Yes I know it was failing before this PR, and isn't even genomicsdb related. Some subworkflow is failing because of another module so I don't think it's within the scope of this PR to fix it

**Comment by muffato** [comment]
I tested on GitPod and the gatk4/genomicsdbimport test works on Docker. Let's merge this PR and open an issue

**Review by muffato — APPROVED** [review]

**Pull request merged**